examples/lighting-app/linux/Options.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "Options.h"
+
+#include <core/CHIPError.h>
+#include <support/CHIPArgParser.hpp>
+
+using namespace chip;
+using namespace chip::ArgParser;
+
+LinuxDeviceOptions LinuxDeviceOptions::mLinuxDeviceOptions;
+
+LinuxDeviceOptions::LinuxDeviceOptions()
+{
+    // Default values
+    mBleDevice = 0;
+}
+
+namespace {
+enum
+{
+    kDeviceOption_BleDevice = 0x1000,
```

Comment:
nit: could we use a `constexpr int kDeviceOption_BleDevice = 0x1000` instead of an anonymous enum?

Provides more typesafety and uses less lines.

Reply:
Prefer to follow the code style of command line arguments in case we need to add more options in the future.